nixos/modules/services/mail/postfix.nix

```diff
@@ -466,22 +466,18 @@ in
         ";
       };
 
-      sslCert = mkOption {
+      clientTlsCAFile = mkOption {
         type = types.str;
-        default = "";
-        description = "SSL certificate to use.";
+        default = "${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt";
+        description = "File containing trusted certification authorities (CA) to verify certificates of mailservers contacted for mail delivery.
+        Defaults to NixOS trusted certification authorities.";
       };
 
-      sslCACert = mkOption {
-        type = types.str;
-        default = "";
-        description = "SSL certificate of CA.";
-      };
-
-      sslKey = mkOption {
-        type = types.str;
-        default = "";
-        description = "SSL key to use.";
+      tlsChainFiles = mkOption {
+        type = types.nullOr (types.listOf types.str);
+        default = null;
+        description = "List of one or more PEM files, each holding one or more private keys directly followed by a corresponding certificate chain.
+        Each key must appear immediately before the corresponding certificate, optionally followed by additional issuer certificates that complete the certificate chain for that key.";
```

Comment:
```suggestion
        description = ''
          List of one or more PEM files, each holding one or more private keys directly followed by a corresponding certificate chain.
          Each key must appear immediately before the corresponding certificate, optionally followed by additional issuer certificates that complete the certificate chain for that key.
        '';
        example = [ "/var/lib/acme/example.com/key.pem" "/var/lib/acme/example.com/cert.pem" ];
```

Reply:
@deviant What don't you like about the suggestion?

Reply:
Apologies, @dotlambda. I was in the middle of writing this out and got waylaid:

The Postfix docs explicitly recommend against splitting up the key and certificate chain for reasons of update atomicity; how about the following?

```
[ "/var/lib/acme/example.com/full.pem" "/var/lib/acme/example.org/full.pem" ]
```

Additionally, the description feels like it's repeating itself a bit, but I'm not sure how to better put it right now.